client/beefy/src/lib.rs

```diff
@@ -96,7 +98,9 @@ where
 	/// Gossip network
 	pub network: N,
 	/// BEEFY signed commitment sender
-	pub signed_commitment_sender: notification::BeefySignedCommitmentSender<B>,
+	pub signed_commitment_sender: BeefySignedCommitmentSender<B>,
+	/// BEEFY best block sender
+	pub beefy_best_block_sender: BeefyBestBlockSender<B>,
```

Comment:
not possible unless we turn from a RPC inquiry method (`beefy_getFinalizedBlockHeader)` into a subscription model like the one exposed for commitments/justifications where the RPC is actually a method to register/unregister caller to said notification stream.

Even then, it might still make sense to have separate exposed notification streams for commitments and round results/finalized blocks.